spec.html

```diff
@@ -10751,28 +10753,34 @@
         </li>
       </ul>
       <emu-grammar>
-        IdentifierReference : `yield`
+        IdentifierReference[Yield, Await] : `yield`
 
-        BindingIdentifier : `yield`
+        BindingIdentifier[Yield, Await] : `yield`
 
-        LabelIdentifier : `yield`
+        LabelIdentifier[Yield, Await] : `yield`
       </emu-grammar>
       <ul>
         <li>
           It is a Syntax Error if the code matched by this production is contained in strict mode code.
         </li>
+        <li>
+          It is a Syntax Error if this production is matched with the Yield grammar parameter set.
```

Comment:
And strictly speaking this restriction only makes sense for `BindingIdentifier` (because `IdentifierRefernce : [~Yield] yield` already ensures `yield` cannot be matched as an IdentifierReference). So maybe add a new early error restriction `<emu-grammar>` block for this case?